vapid.go

```diff
@@ -71,10 +72,15 @@ func getVAPIDAuthorizationHeader(
 		return "", err
 	}
 
+	sub := subscriber
+	if !subIsURL {
```

Comment:
We could also check the url and if it has a protocol then leave it alone otherwise default to `mailto:` so that a new parameter isn't required but otherwise this works just fine.

Reply:
yeah i figured it's a high enough volume use-case that it's worth it not to sniff that out and just pass "true" since it is statically true for us